RELEASE_NOTES.md

```diff
@@ -0,0 +1,38 @@
+# Performance Optimizations
+## Intel Architecture Processors
+  * Improved fp16/bf16 softmax performance with relaxed [accumulation mode](https://oneapi-src.github.io/oneDNN/dev_guide_attributes_accumulation_mode.html#doxid-dev-guide-attributes-accumulation-mode).
+  * Added support and improved perfomance for fp8 matmul with bf16/fp16.
+
+## Intel Graphics Products
+  * Introduced initial optimizations for GPUs based on Xe3 architecture.
+  * Improved performance for convolution for Intel Arc Graphics for Intel Core Ultra processors (Series 2) (formerly Lunar Lake).
+
+## AArch64-based Processors
+
+# Functionality
+  * Introduced support for `select` algorithm in binary primitive. The functionality is optimized for Intel CPUs.
+  * Enabled support for matmul primitive with grouped quantization on weight along N dimension
+  * Introduced support for fp16/bf16 compressed weights in fp32 matmul on Intel CPUs.
+  * Introduced support for grouped scales and zero points in reorder primitive.
+  * Enabled support for 4d weight scale in matmul primitive.
```

Comment:
Not sure since I believe we do not have optimized implementation for these.

```suggestion
  * Enabled support for 4d weight scale in matmul primitive.
  * Graph API: added support for Quantized and non-quantized Gated MLP pattern
```